warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java

```diff
@@ -252,10 +251,12 @@ public void testMaxValueAnyFieldNegAnd() throws Exception {
     public void testMaxIvaratorResultsFailsQuery() throws Exception {
         log.info("------  testMaxIvaratorResultsFailsQuery  ------");
         String regex = RE_OP + "'b.*'";
-        String query = Constants.ANY_FIELD + regex;
+        String city = EQ_OP + "'b-city'";
+        String query = Constants.ANY_FIELD + regex + AND_OP + Constants.ANY_FIELD + city;
```

Comment:
Why is the query changing from `_ANYFIELD_ =~ 'b.*'` to `_ANYFIELD_ =~ 'b.*' && _ANYFIELD_ =~ 'b-city'`? I suspect this fundamentally changes how the query executes..